oas

```diff
@@ -142,6 +142,65 @@ paths:
               schema:
                 $ref: '#/components/schemas/Animal'
 
+  /events:
+    get:
+      summary: Get events for the animals
+      tags:
+        - Animal Information
+      parameters:
+        - name: date_start
```

Comment:
description should state that format is ISO 8601 compliant. Include an example that is consistent with the response example

Reply:
I added an example and extended the description. 
Regarding the docs (https://swagger.io/docs/specification/data-models/data-types/ ) the schema key `"format": "date-time"` means ISO8601

Reply:
You are absolutely right about the specifications but I find that many developers do not read the fine details. I does not hurt to specifically mention ISO 8601.

By the way, I believe your date example has an incorrect format, it should be: 2023-01-01T00:00+00. That applies to all the date examples.

Reply:
I think that ISO also allows shorter presentations. But doesn't hurt to unify all date-examples.
Done.